.changeset/lazy-avocados-whisper.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/i18n": minor
+---
+
+Improved thread metrics to feature user avatars, among other improvements.
```

Comment:
Feel we can explain what the "other improvements" are in the changeset unless they're def not visible by users.